src/vs/workbench/contrib/files/browser/fileCommands.ts

```diff
@@ -682,9 +680,8 @@ KeybindingsRegistry.registerCommandAndKeybindingRule({
 		if (typeof args?.viewType === 'string') {
 			const editorGroupsService = accessor.get(IEditorGroupsService);
 
-			const textInput = editorService.createEditorInput({ options: { pinned: true } });
 			const group = editorGroupsService.activeGroup;
-			await editorService.openEditor(textInput, { override: args.viewType, pinned: true }, group);
+			await editorService.openEditor({ options: { override: args.viewType, pinned: true } }, group);
```

Comment:
I am not entire sure this one works?

Reply:
I'm not sure as well. I don't remember when that code gets called. It looks untitled file specific?